### PR TITLE
Use a unified code style in driver source code

### PR DESCRIPTION
**Pull request by xhaihao**

### CODE REVIEW

**Comment by chivakker** [comment]
@xhaihao can you comment on the plan for this pull request?  As it is touching a good number of files, what is the plan to deploy it?  In particular, how is this standardize with libva and libva-utils?

**Comment by xhaihao** [comment]
@chivakker I will merge the commit soon. Although it is touching lots of files, there is no any change to the functionality.  As for libva/libva-utils, I will do similar changes. BTW a script of style_unify in libva was added years ago.

**Comment by chivakker** [comment]
It is actually a good idea and I look forward to see it merged along with libva and libva-utils

**Comment by xhaihao** [comment]
Rebased and updated commit log, comments.

File: style_unify [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+files=$(find src -name "*.[ch]")
+for i in $files
+do
+    echo $i
+    astyle --style=linux -cnpUH -s4 -M120 $i
```
Comment (ceyusa) [inline style_unify]: I'm very excited about this patches! :+1:

Just a comment about this line, I would recommend to use the long names of the flag, for sake of autodocumentation:

astyle --style=linux --convert-tabs --suffix=none --pad-oper --unpad-paren --pad-header --indent-spaces=4 --max-instatement-indent=120

Comment (xhaihao) [inline style_unify]: Thanks for your comment, I will update the script.

**Pull request merged**